Modules/Revenue/Resources/views/create.blade.php

```diff
@@ -0,0 +1,51 @@
+<div class="modal fade bd-example-modal-lg" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
```suggestion
<div class="modal fade bd-example-modal-lg" id="editRevenueProcced" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Let's change the name to editRevenueProcced or createRevenueProcced